kin-sdk-core/src/androidTest/java/kin/sdk/core/KinAccountTest.java

```diff
@@ -91,6 +92,8 @@ public void exportKeyStore_wrongPassphrase() {
             kinAccount.exportKeyStore("wrongPassphrase", "newPassphrase");
         } catch (PassphraseException e) {
             assertEquals("Wrong passphrase - could not decrypt key with given passphrase", e.getMessage());
+        } catch (OperationFailedException e){
```

Comment:
why to add catch clause? the test should fail on wrong passphrase
I think adding it to `throws` declarations is more accurate.
P.S checking exceptions should be done with `@ExpectedException` annotation instead of catching and asserting (or at least checking the exception type and not message, it's too fragile)